app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::GreetingsController < ApplicationController
+  def index
+    @greetings = Greeting.all
+    render json: @greetings
+  end
+end
```

Comment:
According to the project requirements, kindly note that we are required to create an **API endpoint** that selects a random greeting from your table 🤓🛠

![image](https://github.com/AbdusSattar-70/hello-rails-react/assets/78251927/9072c4be-89f2-4b89-9783-fc838877896a)

e.g.

```ruby
  # GET /messages
  def index
    max = Message.count
    # generate a random number between 1 and max
    random = rand(1..max)
    @messages = Message.where(id: random)

    render json: @messages
  end
```

Please fix that 👍